.github/actions/elixir_cache/action.yml

```diff
@@ -1,19 +1,44 @@
 name: Elixir Cache
 description: Elixir Cache
 
+on:
+  push:
+    branches:
+      - develop
+      #cron data to be provided
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
This is a [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action), it's an action that's re-usable across multiple workflows, we can set a cron job here as the [`on: event` only triggers at the workflow level](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on), let's remove this